blog/2025-01-29-nyc-hackathon/blog.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: nyc-hackathon  # URL slug for the post
+title: "NYC Hackathon"  # Title of the post
+authors: ["DavidSankel"]  # Author(s) of the post
+tags: ["hackathon", "nyc", "blog"]  # Tags for categorization
+---
+
+# NYC Hackathon
+
+![busy coding](20250123_170307.jpg "Several hackathon members busy coding. They give the impression of serious and intense work happening!")
+
+Adobe hosted an exciting and intense half-day Beman hackathon where we worked on
+adding new Beman libraries, addressed infrastructure issues, and brainstormed
+designs for the Beman website. It was awesome to collaborate in person. Here's
+some of the awesome stuff that got accomplished:
+
+- **New Beman tagline: "Tomorrow's C++ Standard Libraries Today"**. Thanks Nick
+  DeMarco for this great contribution!
+- **std::elide** paper implementation from Robert Leahy, a first time
+  contributor!
+- Linus Boehm made some great progress creating a
+  [**std::indirect/std::polymorphic**](https://github.com/bemanproject/indirect/pull/1)
+  Beman library.
+- We had several discussions on the future of the
+  [exemplar](https://github.com/bemanproject/exemplar) library.
+- River Wu identified **root cause of an ASAN hanging issue**.
+- Eddie Nolan and I sketched out a [rough
+  design](https://www.ednolan.com/mockup.html) for the new website.
+
+Overall it was productive and a ton of fun. Thanks to Robert, Eddie, Linus,
+River, Nick, Bret, and Robert for participating!
+
+![portrait of attendees](20250123_170405.jpg "All the hackathon folks standing together in a portrait looking very cool.")
```

Comment:
Is the link weird here -- that is, should there be a paren after the .jpg?

Reply:
This is apparently how it is supposed to be done based on my web searches. The part after the file name is the alt. text.